internal/ocs/validations.go

```diff
@@ -0,0 +1,81 @@
+package ocs
```

Comment:
Would recommend adding an additional directory in the path - internal/operators/<operator-name>
This would make it easier to add more operators in the future.

Reply:
@filanov @avishayt Do we want the ocs related code to be external as `pkg/operators/ocs` or `internal/ocs` .

Reply:
we put code in pkg only if it can be used by other service etc. because this code is relevant to the internal implementation of the service i think that it can be under internal.
It's a good idea to add another directory of operators as @nbalacha suggested.